packages/swayze/lib/src/core/controller/table/header_state.dart

```diff
@@ -79,6 +81,13 @@ class SwayzeHeaderState {
     return result;
   })();
 
+  // TODO: [victor] probably a controller is a better solution
```

Comment:
Shouldnt this be your hoithub username?

Reply:
probably 😆 
But it shouldn't have any TODO when the work is completed.